Frontend/src/Components/CreateListing/InfoSection.jsx

```diff
@@ -19,23 +22,47 @@ const InfoSection = ({
           <option value="" disabled>
             Select a category
           </option>
-          <option value="clothing">Clothing</option>
-          <option value="accessories">Accessories</option>
-          <option value="furniture">Furniture</option>
-          <option value="beauty">Beauty</option>
-          <option value="appliances">Appliances</option>
+          {Object.values(Categories).map((cat) => (
+            <option key={cat} value={cat}>
+              {cat.charAt(0).toUpperCase() + cat.slice(1)}
```

Comment:
so based on my previous comment, for here i would call the getCategoryName(cat) function to display the correct name that you would want to show